Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
-members = ["src/clippy-tracing", "src/cpu-template-helper", "src/firecracker", "src/jailer", "src/log-instrument", "src/log-instrument-macros", "src/rebase-snap", "src/seccompiler", "src/snapshot-editor"]
-default-members = ["src/clippy-tracing", "src/cpu-template-helper", "src/firecracker", "src/rebase-snap", "src/seccompiler", "src/snapshot-editor"]
+members = ["src/acpi-tables", "src/clippy-tracing", "src/cpu-template-helper", "src/firecracker", "src/jailer", "src/log-instrument", "src/log-instrument-macros", "src/rebase-snap", "src/seccompiler", "src/snapshot-editor"]
```

Comment:
Could this just be a module in `vmm`?

Reply:
My ultimate goal is to merge these with the rust-vmm functionality. That's why I was thinking of it more in terms of a crate external to `vmm`, but I don't have a strong preference.

Reply:
I think this is fine, lets just keep track of the upstreaming endeavor somewhere :)

Reply:
I would strongly advice to move this to `vmm`. Otherwise it will become yet another "move crates into vmm" endeavor.